src/App.js

```diff
@@ -108,9 +107,10 @@ class App extends Component {
   // DOM Handlers
   handleClick = (e) => {
     e.preventDefault();
-    if (e.target.id) {
-      this.handleFormToggle(e.target.id);
-    } else return;
+    if (e.target.id) this.handleFormToggle(e.target.id);
+    else if (e.currentTarget.id)
+      this.handleFormToggle(e.currentTarget.id);
+    else return;
```

Comment:
@thomasdahlberg I added e.currentTarget.id because the Material-UI Button component can't access its id with e.target.id.

Reply:
Oh, that's good to know because i was having some issues with switching the buttons over to material-ui